app/scripts/controllers/eventRegistrations.js

```diff
@@ -175,7 +175,13 @@ angular
           $scope.meta = data.meta;
           $scope.registrations = data.registrations;
           $scope.registrants = _.flatten(
-            _.map(data.registrations, 'registrants'),
+            _.map(data.registrations, r => {
+              let regs = r.registrants;
+              for (const reg of regs) {
+                reg.reported = r.reported;
+              }
+              return regs;
+            }),
```

Comment:
Can we be consistent on loop types here? My preference would be vanilla JS map/filter/reduce.

```js
$scope.registrants = data.registrations.flatMap(registration =>
  registration.registrants.map(registrant => ({
    ...registrant,
    reported: registration.reported
  }))
);
```

Reply:
I would also prefer not to clone and denormalize data as it tends to lead to bugs and state inconsistencies. Idk if there's a good way to look this up though... You could do `$scope.registrations[registrant.registrationId].reported`... As you don't use this very often in the template, I think looking it up when you need it would be doable.

Reply:
Reverted this change, added `isRegistrantReported` function instead